### PR TITLE
Add support for UICollectionViewCell

### PR DESCRIPTION
**Pull request by pyankoff**
Add support for UICollectionViewCell

### CODE REVIEW

**Comment by edwardsdl** [comment]
I was about to submit a PR for the very same feature! :+1:

File: NibDesignable.swift [hunk 1]
```diff
@@ -137,3 +137,33 @@ public class NibDesignableControl: UIControl {
         self.addConstraints(NSLayoutConstraint.constraintsWithVisualFormat("V:|[view]|", options:NSLayoutFormatOptions(rawValue: 0), metrics:nil, views: bindings))
     }
 }
+
+@IBDesignable
+public class NibDesignableCollectionViewCell: UICollectionViewCell {
+    
+    // MARK: - Initializer
+    override init(frame: CGRect) {
```
Comment (edwardsdl) [inline NibDesignable.swift]: This should be `override public init(frame: CGRect) {`

Comment (pyankoff) [inline NibDesignable.swift]: That's right, fixed, thanks!

**Comment by mbogh** [comment]
Thanks :)

Still bugging me with the non-DRY-ness of the code. But won't hold this back :)

**Pull request merged**